### ISSUE TITLE
detecting_overlapped_peak_colocalization (got idea)

### ISSUE THREAD

**Issue by jerryShad** [polimi911/MSPC-Package#9]
to perform co-localization for peak the from multiple genes; following must be done:
1. first, implement function which load reference gene in R and assign all its peak regions to interval-tree; 
   Question: if interval-tree report intersected peak regions from these gene, how can I retrieve back its p-value and peak name, and located which chromosome; THINK HARDER
2. second, use apply family function to process already classified peak from initial peak classification (where peak may classified as stringent or weak peak); SO I want to carefully design the main body of this apply function that peak up every peak, ask findOverlap fucntion to report is there any overlapped region from selected reference gene. 

TO SUM UP:
apply(peak, 2, function() )             ## take each peak and ask its overlapped region from intervaltree

function for loading reference gene and assign all its peak regions to intervaltree

function for report which overlapped peak is, what is p-value ....
